src/test/java/org/jenkinsci/plugins/oic/PluginTest.java

```diff
@@ -822,6 +846,14 @@ private static void assertTestUserEmail(User user) {
                 "Email should be " + TEST_USER_EMAIL_ADDRESS);
     }
 
+    private static void assertTestAvatar(User user, WireMockExtension wireMock) {
+        String expectedAvatarUrl = "http://localhost:%d/my-avatar.png".formatted(wireMock.getPort());
+        OicAvatarProperty avatarProperty = user.getProperty(OicAvatarProperty.class);
+        assertEquals(expectedAvatarUrl, avatarProperty.getAvatarUrl(), "Avatar url should be " + expectedAvatarUrl);
+        assertEquals("Openid Connect Avatar", avatarProperty.getDisplayName());
+        assertNull(avatarProperty.getIconFileName(), "Icon filename must be null");
```

Comment:
should this not check the AvatarResolver also?

e.g. (untested)

```suggestion
        assertEquals(expectedAvatarUrl, avatarProperty.getAvatarUrl(), "Avatar url should be " + expectedAvatarUrl);
        assertEquals("OpenID Connect Avatar", avatarProperty.getDisplayName());
        assertNull(avatarProperty.getIconFileName(), "Icon filename must be null");
        String urlViaAvatarResolver = UserAvatarResolver.resolve(user, "48x48");
        assertEquals(expectedAvatarUrl, urlViaAvatarResolver, "Avatar url should be " + expectedAvatarUrl);
```